robot/impl/resource_manager_test.go

```diff
@@ -1838,13 +1838,19 @@ func TestReconfigureParity(t *testing.T) {
 		"data/diff_config_deps11.json",
 		"data/diff_config_deps12.json",
 	}
-	logger := logging.NewTestLogger(t)
 	ctx := context.Background()
 
 	testReconfigureParity := func(t *testing.T, initCfg, updateCfg string) {
 		name := fmt.Sprintf("%s -> %s", initCfg, updateCfg)
 		t.Run(name, func(t *testing.T) {
 			t.Parallel()
+			// Capture logs for this sub-test run. Only output the logs if the test fails.
+			logger := logging.NewInMemoryLogger(t)
```

Comment:
Very cool; do we want to broadcast the availability of this type of logger to engineering? Probably helpful for other tests.

Reply:
We can, though I'm not interested in going through my fretting process in how to craft that message today.

But I think if we want to actually see adoption, the best way to get that is to still do some legwork to identify tests that could use this and send a message to a test maintainer with this example.